data-access/src/app/domain/contexts/cases/service-ticket/v1/service-ticket.value-objects.ts

```diff
@@ -10,7 +9,16 @@ export const StatusCodes = {
   Closed: 'CLOSED',
 };
 
-export class Title extends VOString({ trim: true, maxLength: 200, minLength: 5 }) { }
-export class Description extends VOString({ trim: true, maxLength: 2000 }) { }
-export class StatusCode extends VOSet(Object.values(StatusCodes)) { }
-export class Priority extends VOInteger({ min: 1, max: 5 }) { }
\ No newline at end of file
+export class Title extends VOString({ trim: true, maxLength: 200, minLength: 5 }) {}
+export class Description extends VOString({ trim: true, maxLength: 2000 }) {}
+export class StatusCode extends VOSet(Object.values(StatusCodes)) {}
+export class Priority extends VOInteger({ min: 1, max: 5 }) {}
+export class AssignedVendor extends VOObject({
```

Comment:
Remove this value object for AssignedVendor, we don't need value objects for reference fields